android/src/main/java/com/onegini/mobile/sdk/reactnative/handlers/customregistration/SimpleCustomTwoStepRegistrationActionImpl.kt

```diff
@@ -29,11 +28,20 @@ class SimpleCustomTwoStepRegistrationActionImpl(private val idProvider: String)
         return idProvider
     }
 
-    override fun returnSuccess(result: String?) {
-        calback?.returnSuccess(result)
+    override fun returnSuccess(result: String?): Boolean {
+        return callback?.let { customRegistrationCallback ->
+            customRegistrationCallback.returnSuccess(result)
+            callback = null
+            true
+        } ?: false
     }
 
-    override fun returnError(exception: Exception?) {
-        calback?.returnError(exception)
+    override fun returnError(exception: Exception?): Boolean {
+        callback?.let { customRegistrationCallback ->
+            customRegistrationCallback.returnError(exception)
+            callback = null
+            return true
+        }
```

Comment:
in both `returnErrors` you miss our friend Elvis

Reply:
This will be fixed in a merge with https://github.com/onewelcome/sdk-react-native/pull/87/